data/test_rag.py

```
@@ -0,0 +1,21 @@
+import os
+from query_data import query_rag
```

Ruff (0.8.2)

`query_data.query_rag` imported but unused

Remove unused import: `query_data.query_rag`

(F401)